script/brave_license_helper.py

```diff
@@ -33,6 +33,9 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         # android_deps/libs instead and it's special-cased further down.
         os.path.join('brave', 'third_party', 'android_deps'),
 
+        # No third-party code directly under ios_deps.
```

Comment:
I think we need to do something like we did for Android deps here (see the `# Add all Android libraries since they're not directly contained within a third_party directory.` block below). That way the licensing script will pick up any new third-party that was added without licensing info.

It doesn't have to block this PR necessarily, but at the very least, we should have an issue filed and a `FIXME` here.

Reply:
Fixed in https://github.com/brave/brave-core/pull/21553/commits/d05f0c407975d4c34d611641a675edf026730a4d

One thing I had to do was unfortunately not walk recursively, because unfortunately Jitsi includes a `node_modules` folder inside of their frameworks and it gets wonky… I can swap it up to walk recursively the same way as Android but we'd need to then exclude their node_modules directory for each framework in the xcframework. Instead I made a note that we'd never nest deps for now